build.zig

```diff
@@ -14,7 +16,8 @@ pub fn build(b: *std.Build) void {
             .optimize = optimize,
         });
 
-        exe.addModule("swayipc", swayipc);
+        exe.root_module.addImport("swayipc", swayipc);
+        // exe.addModule("swayipc", swayipc);
```

Comment:
Any particular reason why this is a comment?

Reply:
Just an oversight...I commented it out when updating the `addImport` call and forgot to delete it.